File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2059.cpp
#include <iostream>

using namespace std;

int main() {
    int p, j1, j2, r, a;
    cin >> p >> j1 >> j2 >> r >> a;
    if(r==1){ // j1 roubou
        if(a==1){
            cout << "Jogador 2 ganha!" << endl;
        } else{
            cout << "Jogador 1 ganha!" << endl;
        }
    } else{ // j1 nao robou
        if(a==1){
            cout << "Jogador 1 ganha!" << endl;
        } else{
            if((j1+j2)%2==0){
                if(p==1){
                    cout << "Jogador 1 ganha!" << endl;
                } else{
                    cout << "Jogador 2 ganha!" << endl;
                }
            } else{
                if(p==1){
                    cout << "Jogador 2 ganha!" << endl;
                } else{
                    cout << "Jogador 1 ganha!" << endl;
                }
            }
        }
    }
    
    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2059.lua
p = io.read("*number")
j1 = io.read("*number")
j2 = io.read("*number")
r = io.read("*number")
a = io.read("*number")

if(r==1) then
    if(a==1) then
        print("Jogador 2 ganha!")
    else
        print("Jogador 1 ganha!")
    end
else
    if(a==1) then
        print("Jogador 1 ganha!")
    else
        if((j1+j2)%2==0) then
            if(p==1) then
                print("Jogador 1 ganha!")
            else
                print("Jogador 2 ganha!")
            end
        else
            if(p==1) then
                print("Jogador 2 ganha!")
            else
                print("Jogador 1 ganha!")
            end
        end
    end

end